src/app/personside/dialogpanel/fortsettDialog/NyFortsettDialog.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import { FormEvent } from 'react';
+import { AlertStripeInfo } from 'nav-frontend-alertstriper';
+import { Traad, TraadType } from '../../../../models/meldinger/meldinger';
+import TidligereMeldinger from './tidligereMeldinger/TidligereMeldinger';
+import TekstFelt from '../sendMelding/TekstFelt';
+import { UnmountClosed } from 'react-collapse';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { DialogpanelFeilmelding, FormStyle } from '../fellesStyling';
+import KnappMedBekreftPopup from '../../../../components/KnappMedBekreftPopup';
+import BrukerKanSvare from './BrukerKanSvare';
+import styled from 'styled-components/macro';
+import { FortsettDialogValidator } from './validatorer';
+import { DialogPanelStatus, FortsettDialogPanelState, FortsettDialogState } from './FortsettDialogTypes';
+import { erJournalfort } from '../../infotabs/meldinger/utils/meldingerUtils';
+import { capitalizeName } from '../../../../utils/string-utils';
+import persondataResource from '../../../../rest/resources/persondataResource';
+import { Checkbox } from 'nav-frontend-skjema';
+
+const SubmitKnapp = styled(Hovedknapp)`
+    white-space: normal;
+`;
+
+const StyledKnappMedBekreftPopup = styled(KnappMedBekreftPopup)`
+    width: 100%;
+`;
+
+const Margin = styled.div`
+    /* Pga React Collapse må vi slenge på noen div'er som tar seg av marginer for å unngå hopp i animasjon */
+`;
+
+const StyledAlertStripeInfo = styled(AlertStripeInfo)`
+    margin-top: 1rem;
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+    margin-top: 0.1rem;
+`;
```

Comment:
[NITPICK] typisk forsøker man å holde seg til `margin-bottom` og `margin-right`. 
For `margin-bottom` finnes det ett sett av utility-klasser; `blokk-m, blokk-xs` etc. som kan bruker.

Årsaken til dette er at det da er enklere å holde styr på hvor marginer kommer fra, siden `margin-top` og `margin-bottom` mellom to komponenter vil kollapse, og resultere i at avstanden bare blir den største av de.